c2corg_ui/static/js/documentediting.js

```diff
@@ -185,21 +186,20 @@ app.DocumentEditingController = function($scope, $element, $attrs,
    */
   this.api_ = appApi;
 
-  // allow association only new outing  to existing route
+  // allow association only new outing to existing route
   if (this.ngeoLocation_.hasParam('routes')) {
     this.urlParams_ = {'routes': this.ngeoLocation_.getParam('routes')};
     this.pushDocToAssociations_();
   }
 
-  // When creating a new document, the model is not created until
-  // the form is touched. At least create an empty object.
-  this.scope_[this.modelName_] = this.scope_['document'] = this.documentService.document;
```

Comment:
removing this line causes the impossibility to select orientations in the SVG Orientation selector, because the SVG uses `document.orientation` syntax and now it throws an error `TypeError: Cannot read property 'orientations' of undefined`